src/server/test/unit/LineReader.test.ts

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-var */
+/*!
```

Comment:
stick with the 2 header formats we already have, don't need a 3rd variation; see e.g. src\server\telemetry\ServerTelemetry.ts or src\client\test\unit\CliAcquisition.test.ts

Also, move the eslint exception comments below the header

Reply:
done